src/index.js

```diff
@@ -25,6 +26,8 @@ require('./routes/routes').attachRoutes(router)
 app.use(router)
 app.use(noPathHandler)
 app.use(errorHandler)
+app.use(passport.initialize())
+require('./middlewares/passport-config')(passport)
```

Comment:
pretty sure these two lines go right after the `app.use(reqLogger)` line, since passport needs to "intercept" every request that comes through before they go to routes